examples/scenarios/wefe_pv_panel/scripts/infer.py

```diff
@@ -26,4 +28,15 @@
     **kwargs,
 )
 
-# run building.infer, reopen json file and ammend datapackage
\ No newline at end of file
+# this function adds any additional foreign keys to the datapackage.json file
+otp_building.add_foreign_keys_to_datapackage(
+    "datapackage.json",
+    {
+        "pv_panel": {
+            "from_bus": {"resource": "bus", "fields": "name"},
+            "to_bus": {"resource": "bus", "fields": "name"},
+            "t_air": {"resource": "pv_panel_profile"},
```

Comment:
Looking at all values of the `name` field of all `schema.fields` field for all resources which have `data/sequences` as value of their `path` field let us find the information necessary to infer `"t_air": {"resource": "pv_panel_profile"},` as well

Reply:
ah yeah that makes sense, so have it e.g. like:
```        "pv_panel": {
            "from_bus": {"resource": "bus", "fields": "name"},
            "to_bus": {"resource": "bus", "fields": "name"},
            "t_air": {"fields": "name"}, ```